### PR TITLE
Add quickstart for Azure and update AWS to use new pre-reqs and cdp modules

### PR DESCRIPTION
**Pull request by jimright**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +74,28 @@ ingress_extra_cidrs_and_ports = {
...
+### Sample Configuration file for Azure
+
+```yaml
+# ------- Global settings -------
+env_prefix = "<ENTER_VALUE>" # Required name prefix for cloud and CDP resources, e.g. cldr1
+
+# ------- Cloud Settings -------
+azure_region = "<ENTER_VALUE>" # Change this to specify Cloud Provider region, e.g. westeurpoe
+
+public_key_text = "<ENTER_VALUE> # Change this with the SSH public key text, e.g. ssh-rsa AAA....
```
Comment (wmudge) [inline README.md:86]: Missing quote

File: azure/terraform.tfvars.template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ------- Global settings -------
+env_prefix = "<ENTER_VALUE>" # Required name prefix for cloud and CDP resources, e.g. cldr1
+
+# ------- Cloud Settings -------
+azure_region = "<ENTER_VALUE>" # Change this to specify Cloud Provider region, e.g. westeurpoe
+
+public_key_text = "<ENTER_VALUE> # Change this with the SSH public key text, e.g. ssh-rsa AAA....
```
Comment (wmudge) [inline azure/terraform.tfvars.template:21]: Missing quote

**Review by wmudge — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jimright) [reply README.md]: Resolved in [4eaae82](https://github.com/cloudera-labs/cdp-tf-quickstarts/pull/4/commits/4eaae82794b9d596b1b002fbb68323bd5335c71e)

File: azure/terraform.tfvars.template (hunk 2 above)
Comment (jimright) [reply azure/terraform.tfvars.template]: Resolved in [4eaae82](https://github.com/cloudera-labs/cdp-tf-quickstarts/pull/4/commits/4eaae82794b9d596b1b002fbb68323bd5335c71e)

**Pull request merged**